x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -179,7 +185,9 @@ export const loadPrebuiltDetectionRules = () => {
  * load prebuilt rules by clicking button on page header
  */
 export const loadPrebuiltDetectionRulesFromHeaderBtn = () => {
-  cy.get(LOAD_PREBUILT_RULES_ON_PAGE_HEADER_BTN).click().should('not.exist');
+  cy.log('load prebuilt detection rules from header');
+  waitTillPrepackagedRulesReadyToInstall();
```

Comment:
let's use one name for prepackaged/prebuilt rules.
In some places it's prebuilt, in other prepackaged. Can be confusing, especially for new developers

Reply:
That's been already discussed: https://github.com/elastic/kibana/pull/142311#discussion_r988149034